src/component/Logo_SignUp.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image} from 'react-native';
+import { render } from 'react-dom';
+
+export default function Logo() {
+    return(
+        <View style={styles.container}>
+            <Image style={{width:110, height:120}}
+                source={require('../../assets/garuda.png')}/>
+            <Text style={styles.LogoText}>
+                Hello!
+            </Text>
+            <Text style={styles.LogoText_2}>
+                Please, Sign Up to Continue
+            </Text>
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    container: {
```

Comment:
change `container` to `Container`. Make convention for variable naming, and stick with it